### ISSUE TITLE
Magento 2.4.5 Compatibility

### ISSUE THREAD

**Issue by xpoback** [punchout-catalogs/magento2-cookie2#2]
We've tried the module on Magento Commerce 2.4.5 and got an error on any component or script what required `jquery/jquery-storageapi` like `accordion` or `collapsible`:

```main.js:45 Error: Script error for "jquery/jquery.storageapi.min", needed by: jquery/jquery-storageapi
https://requirejs.org/docs/errors.html#scripterror
    at makeError (require.js:168:17)
    at HTMLScriptElement.onScriptError (require.js:1738:36)
```
We suspect there is some incompatibility with this Magento version (which was released August). 
Could you please check and confirm it's compatible with Magento 2.4.5?

**Comment by punchoutcatalogs** [comment]
Fixed in the latest version 0.3.1.

**Comment by punchoutcatalogs** [comment]
The issue was Magento 2.4.5 changes related to `jquery/jquery-storageapi` - M2 team simply started to use another library.
